WELCOME.md

```diff
@@ -0,0 +1,47 @@
+
+Need to report Code of Conduct violation? The admins are listed at the bottom of the Code of Conduct. Snap a screenshot, report it.
+
+🚴🏿‍♀️🕺🏻👩‍💻😻⚾️✈️📺 Second, you've automagically been added to the #introductions channel. Please introduce yourself to the community there and share something fun about yourself. Where you're from, what you're working on, what your most peculiar hobby is, what your favorite color is, etc.. We want to get to know you! Also, there's probably a channel for that.
+
+📣 Speaking of channels. We have a lot. When thinking about where to post a message, go through the channel list and check there first. It’s very tempting with ~600 eyeballs to post something to #general. We tend to treat #general like a watercooler, not a place to garner attention.
+
+_Popular channels:_
+
+- #frontend - talk about your javascript
+- #devs-n-ops - do not talk about your javascript
+- #cohordeworking - when, where, why, who
+- #business - general business topics
+- #mental-health - talk about it
+- #gigs - the ONLY place you can post/ask for jobs/gigs. They should be 100% available to Minnesota with no filter criteria, ala “might be interested in remote” Many people have notifications on this channel, so 🧵any conversations beyond the initial post.
+
+_Fun, small channels:_
+
+- #art-hackers
+- #biking
+- #awww
+- #pets
+- #sports
+- #outdoors
+
+🦠 And then there’s COVID-19, the big 🐘 in the room. #general is a COVID-19 free zone. This is a **strict** policy and we will actively police it. While we understand it’s a hard time for everyone, you can’t leave #general and we want people to have a refuge as it is impacting many of us indirectly or directly.
+
+SO with that:
+#covid19-discuss - general discussion of COVID-19 related topics
+#covid19-reference - fact based information sharing (links only, move discussions to #covid19-discuss)
+#covid19-memes - the ONLY place to allow covid related memes (that also adhere to the Code of Conduct)
+
+:spam: Spam posts, sales-y posts, self-promoting without community participation is not allowed. Admins will ask posts like this to be removed or moved to a different channel. If not done by the original poster within 15 min, we will do it for you. Don't make us do it for you.
+
+💌 Want to invite someone? Head over to #invites and kindly answer these questions:
+- Who are you inviting? (name and email)
+- Do they currently reside in Minnesota?
+- Describe how they are tech adjacent or tech in general.
+- Are they expecting this invite? We only invite people who have opted into this invite, otherwise we're obnoxious.
+_Note:_ Please invite good humans that will read and abide by the Code of Conduct. You're using _your_ social capital to say "this is a good human."
```

Comment:
This is awesome

Reply:
The first sentence is @whoisjake's. The second is mine 😊

Reply:
They're both great, and both important!

Reply:
I like it.